src/pybind/mgr/nfs/ganesha_conf.py

```diff
@@ -10,6 +10,42 @@
     from nfs.module import Module
 
 
+def _indentation(depth: int, size: int = 4) -> str:
+    return " " * (depth * size)
+
+
+def _format_val(block_name: str, key: str, val: str) -> str:
+    if isinstance(val, list):
+        return ', '.join([_format_val(block_name, key, v) for v in val])
+    if isinstance(val, bool):
+        return str(val).lower()
+    if isinstance(val, int) or (block_name == 'CLIENT'
+                                and key == 'clients'):
+        return '{}'.format(val)
+    return '"{}"'.format(val)
```

Comment:
(nit) the diff is somewhat hard to read when these are defined at the top and the `format_block` is at the bottom, should we move these down so all the related funcs are located together?

Reply:
Thanks. I will keep that in mind for next time.